frontend/README.md

```diff
@@ -26,6 +26,21 @@ You can choose your own preferred webserver like for example `nginx`, `serve` or
    npm run dev
    ```
 
+## Setting up the environment variables
+
+The project requires a couple of environment variables to run, if you want to develop on this codebase.
```

Comment:
Don't really know what the if you want to develop on this codebase does here, it always needs environment variables to run?

Reply:
Exactly, not project, but application